nifi-commons/nifi-hadoop-utils/src/test/java/org/apache/nifi/hadoop/TestKerberosProperties.java

```diff
@@ -19,25 +19,20 @@
 import org.apache.hadoop.conf.Configuration;
 import org.apache.nifi.components.ValidationResult;
 import org.apache.nifi.logging.ComponentLog;
-import org.apache.nifi.util.NiFiProperties;
 import org.junit.Assert;
 import org.junit.Test;
 import org.mockito.Mockito;
 
 import java.io.File;
 import java.util.List;
 
-import static org.mockito.Mockito.when;
-
 public class TestKerberosProperties {
 
     @Test
     public void testWithKerberosConfigFile() {
         final File file = new File("src/test/resources/krb5.conf");
-        final NiFiProperties niFiProperties = Mockito.mock(NiFiProperties.class);
-        when(niFiProperties.getKerberosConfigurationFile()).thenReturn(file);
 
-        final KerberosProperties kerberosProperties = KerberosProperties.create(niFiProperties);
+        final KerberosProperties kerberosProperties = new KerberosProperties(file);
         Assert.assertNotNull(kerberosProperties);
```

Comment:
I think there were some weird things happening during the transition from static initialization so these are sanity checks.